code/game/machinery/kitchen/cooking_machines/oven.dm

```diff
@@ -7,19 +7,19 @@
 	appliancetype = OVEN
 	food_color = "#A34719"
 	can_burn_food = 1
-	active_power_usage	= 19000
+	active_power_usage = 4 KILOWATTS
 	//Based on a double deck electric convection oven
 
-	resistance = 72000
-	idle_power_usage	= 6000
+	resistance = 10000
+	idle_power_usage = 6000
```

Comment:
Takes 4KW to heat up and 6KW to keep the temperature ?
Dont just go to 20% of the initial value when changing stats of a item.
Use smaller steps and repeat until finding the optimal value.